search-sentiments-map/routes/search.js

```diff
@@ -233,9 +241,10 @@ async function updateSearchResultsForTopic(query) {
     // Use 100 queries per minute limit for the Custom Search API, and include
     // a pause of 1 minute to prevent surpassing limit.
     if (i !== 0 && i % 100 === 0) {
-      await new Promise(resolve => setTimeout(resolve, 60000));
+      await new Promise(resolve =>
+          setTimeout(resolve, PAUSE_TO_PREVENT_REACHING_QUOTA_1_MIN_MS));
     }
-    // Update {@code countryData} within the functions called.
+    // Update `countryData` within the functions called.
```

Comment:
Keep the @code?

Reply:
I just looked this up, apparently @code is deprecated. Will swap this and all previous/future comments on this to backticks.
https://google.github.io/styleguide/jsguide.html#appendices-documentation-annotations